tests/Functional/ClientTest.php

```diff
@@ -116,7 +115,6 @@ function (Psr7Request $request) use ($registrationNumber): Psr7Response {
 
         $response = $this->fixture->vehicles()->enquireDetails($request);
 
-        $this->assertInstanceOf(EnquiryResponse::class, $response);
```

Comment:
These `assertInstanceOf` calls are covered by PHPStan and return type declarations.